javascript/new-src/Gruntfile.js

```diff
@@ -108,6 +108,11 @@ module.exports = function(grunt) {
         verbose: true
       }
     },
+    stylelint: {
+      all: [
+        'css/*.css'
```

Comment:
You've made changes to the HTML with stylelint enable/disable comments, but it's not being checked here, should it be?

Reply:
There is actually no HTML file inside the `javascript/new-src/` directory. I manually tested the old webinterface with the following command (as mentioned in the commit description):

```bash
npx stylelint "olad/www/*.html" --config="javascript/new-src/.stylelintrc.json" --config-basedir="javascript/new-src/"
```

Reply:
Ah yes, I think I remember this discussion before.

Could we add a note somewhere, maybe in this file, explaining how to check the other files.

Would you mind just running it against the RDM stuff too:
https://github.com/OpenLightingProject/ola/tree/master/tools/rdm/static

Reply:
@peternewman I just included the files from the other directories in the Grunt task now, so they are also automatically tested by Travis. I hope this is fine for you; otherwise I can revert the change and add a comment as suggested above.

There were actually a few issues in the RDM directory, which I fixed (or disabled the respective Stylelint rule where it makes sense).

Reply:
Thanks @FloEdelmann . I guess ideally it would be nice to keep any linting local to each folder, but I guess it's probably not worth the overheads.